cluster-management/setup/configuring-date-and-timezone/index.md

```diff
@@ -20,16 +24,72 @@ ntpd.service - Network Time Service
            └─483 /usr/sbin/ntpd -g -n -u ntp:ntp -f /var/lib/ntp/ntp.drift
 ```
 
+[681.0.0]: https://coreos.com/releases/#681.0.0
+[ntp.org]: http://ntp.org/
+[systemd-timesyncd]: http://www.freedesktop.org/software/systemd/man/systemd-timesyncd.service.html
+
+## Switching between systemd-timesyncd and ntpd
+
+If you are on CoreOS 681.0.0 or later you can switch back to the classic ntpd
+with the following commands or cloud config:
+
+```sh
+sudo systemctl stop systemd-timesyncd
+sudo systemctl mask systemd-timesyncd
+sudo systemctl enable ntpd
+sudo systemctl start ntpd
+```
+
+```yaml
+#cloud-config
+
+coreos:
+  units:
+    - name: systemd-timesyncd.service
+      command: stop
+      mask: true
```

Comment:
Can we add a comment on to here that describes what its doing? I'm thinking of a shorter version of this:

```
mask: true    #mask will ensure that this unit isn't started if it was previously enabled
```